data-imports/import-aep.py

```diff
@@ -33,10 +33,10 @@ def main():
 
 def import_csv(args):
     log.info('Importing pre-created AEP csv...')
-    data_file = 'data/AEP_LIST.csv'
+    data_file = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../data/AEP_LIST.csv'))
```

Comment:
OK, this incantation should build out an absolute path (allowing the script to be run from anywhere) to the AEP file.

As long as the directory structure is consistent, this should work in both Docker and elsewhere.